src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir , serif;
+  font-weight: 300;
+}
+
+/* #region form */
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+/* #endregion */
+
+/* #region form__input */
+.form__input {
+  height: 70px;
+  padding: 25px 62px;
+  font-size: 16px;
+  line-height: 22px;
+
+  font-family: inherit;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.form__input--small {
+  height: 42px;
+  padding: 12px 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+/* #endregion */
+
+/* #region icon */
+.form__icon {
+  position: absolute;
+
+  top: 25px;
+  left: 26px;
+  height: 19px;
+}
+
+.form__icon--small {
+  top: 15px;
+  left: 13px;
+  height: 11px;
+}
+/* #endregion */
+
+/* placeholder */
```

Comment:
pseudo-element generally explains itself - no need for extra comments in such case
same for `:hover` and `:focus` pseudo-classes - don't over-comment
it's a matter of balance 😉